README.md

```diff
@@ -124,28 +124,36 @@ parser](docs/custom_sql_parser.md).
 ## Contributing
 
 Contributions are highly encouraged! However, the bandwidth we have to
-maintain this crate is fairly limited.
+maintain this crate is limited. Please read the following sections carefully.
 
-Pull requests that add support for or fix a bug in a feature in the
-SQL standard, or a feature in a popular RDBMS, like Microsoft SQL
+### New Syntax
+
+The most commonly accepted PRs add support for or fix a bug in a feature in the
+SQL standard, or a a popular RDBMS, such as Microsoft SQL
 Server or PostgreSQL, will likely be accepted after a brief
-review.
+review.  Any SQL feature that is dialect specific should be parsed by *both* the relevant [`Dialect`] 
+as well as the omnibsus [`GenericDialect`].
```

Comment:
```suggestion
as well as the omnibus [`GenericDialect`].
```

Reply:
This word may be confusing to non-native English speakers.

Reply:
Good call -- I think it is not necessary in this context. 

```suggestion
as well as [`GenericDialect`].
```